src/Components/Ignitor/src/Ignitor.csproj

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.0</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <IsShippingPackage>false</IsShippingPackage>
```

Comment:
If the decision is to make this into a product asset, please ensure the final (squashed) commit says "Make Ignitor a **non**-shipping package"